utam-compiler/src/main/java/utam/compiler/helpers/TranslationContext.java

```diff
@@ -246,7 +240,7 @@ public ElementContext getElement(String name) {
       return DOCUMENT_ELEMENT;
     }
     if (!elementContextMap.containsKey(name)) {
-      throw new UtamError(String.format(ERR_CONTEXT_ELEMENT_NOT_FOUND, name));
+      throw new NullPointerException();
```

Comment:
Is `NullPointerException` the correct choice here? Nothing is really "null," just that the key wasn't found in the map.

Reply:
addressed by adding error code as a parameter and throwing compiler error